--- origsrc/pvoc-0.1.12/Makefile	2008-01-15 04:59:12.000000000 -0600
+++ src/pvoc-0.1.12/Makefile	2009-07-12 03:24:23.045070200 -0500
@@ -1,8 +1,9 @@
 PREFIX = /usr/local
 
-OPTS = -O6 -ffast-math -funroll-loops -Wall -fPIC
+OPTS = $(CXXFLAGS) -O6 -ffast-math -funroll-loops -Wall
 
 CC = g++
+STRIP = strip
 
 PLUG = pvoc
 UTIL = stretch
@@ -47,7 +48,7 @@ run: all
 	sudo python -i test.py
 
 $(PLUG).so: $(PLUGOBJ) $(PVOCOBJ)
-	$(CC) -nostartfiles -shared -o $@  $(PLUGOBJ) $(PVOCOBJ) $(PLUGLDOPTS) 
+	$(CC) -shared -o $@  $(PLUGOBJ) $(PVOCOBJ) $(PLUGLDOPTS) 
 
 $(UTIL): $(UTILOBJ) $(PVOCOBJ)
 	$(CC) -o $@ $(UTILOBJ) $(PVOCOBJ) $(UTILLDOPTS)
@@ -63,8 +64,8 @@ tags: $(SOURCES) $(HEADERS)
 	@-if [ -x /usr/bin/ctags ]; then ctags $(SOURCES) $(HEADERS) ; fi
 
 install: all
-	strip $(PLUG).so
-	strip $(UTIL)
+	$(STRIP) $(PLUG).so
+	$(STRIP) $(UTIL)
 	install -d $(PLUGDEST)
 	install -d $(UTILDEST)
 	install -d $(MAN1DEST)
